mvc/src/main/java/nextstep/mvc/controller/tobe/AnnotationHandlerMapping.java

```diff
@@ -22,9 +28,46 @@ public AnnotationHandlerMapping(final Object... basePackage) {
 
     public void initialize() {
         log.info("Initialized AnnotationHandlerMapping!");
+
+        for (final Object eachPackage : basePackage) {
+            final Reflections reflections = new Reflections(eachPackage);
```

Comment:
```suggestion
       Reflections reflections = new Reflections(basePackage);
```
한번에 패키지 루트들을 지정할 수 있어요!